src/components/DietContainer/DietContainer.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import './DietContainer.scss';
+
+const DietContainer = ({ ExerciseData, mealTime, iconImg }) => {
+  return (
+    <div className="DietContainer">
+      <h2 className="breakfastText timeText">
+        <span className="iconTime">
+          <img src={iconImg} />
+        </span>
+        &nbsp;
+        {mealTime}
+      </h2>
+      <ul className="dietBoxWrapper">
+        {ExerciseData &&
+          ExerciseData.map((result) => (
```

Comment:
- optional chaining하시면 됩니다.
- PascalCase는 주로 컴포넌트 네이밍에 사용됩니다. camelCase로 수정해주세요!
```suggestion
        {exerciseData?.map((result) => (
```